lib/box/srv.go

```diff
@@ -340,6 +340,23 @@ func getLibcontainerConfig(containerID, rootfs string, cfg Config) (*configs.Con
 				Destination: "/dev/kmsg",
 				Flags:       syscall.MS_BIND,
 			},
+			// /run has to be mounted explicitly as tmpfs in order to be able
+			// to mount /run/udev below
+			{
+				Source:      "tmpfs",
+				Destination: "/run",
+				Device:      "tmpfs",
+				Flags:       syscall.MS_NOSUID | syscall.MS_NODEV,
+				Data:        "mode=755",
+			},
+			// /run/udev is used by OpenEBS node device manager to detect
+			// added and removed block devices
+			{
+				Device:      "bind",
+				Source:      "/run/udev",
```

Comment:
Hmm, I've seen errors from `systemd-udevd` related to this in robotest logs:
```
-- Logs begin at Fri 2019-12-06 04:36:50 CET, end at Fri 2019-12-06 04:38:43 CET. --
Dez 06 04:36:56 robotest-52efe7d3-node-0 systemd[1]: Starting udev Kernel Device Manager...
Dez 06 04:36:56 robotest-52efe7d3-node-0 systemd[1]: Started udev Kernel Device Manager.
Dez 06 04:36:56 robotest-52efe7d3-node-0 systemd-udevd[571]: could not unlink /run/udev/queue: No such file or directory
Dez 06 04:36:56 robotest-52efe7d3-node-0 systemd-udevd[571]: could not unlink /run/udev/queue: No such file or directory
Dez 06 04:36:56 robotest-52efe7d3-node-0 systemd-udevd[571]: could not unlink /run/udev/queue: No such file or directory
Dez 06 04:36:56 robotest-52efe7d3-node-0 systemd-udevd[571]: could not unlink /run/udev/queue: No such file or directory
Dez 06 04:36:56 robotest-52efe7d3-node-0 systemd-udevd[571]: could not unlink /run/udev/queue: No such file or directory
Dez 06 04:36:56 robotest-52efe7d3-node-0 systemd-udevd[571]: could not unlink /run/udev/queue: No such file or directory
Dez 06 04:36:57 robotest-52efe7d3-node-0 systemd-udevd[571]: could not unlink /run/udev/queue: No such file or directory
Dez 06 04:36:57 robotest-52efe7d3-node-0 systemd-udevd[571]: could not unlink /run/udev/queue: No such file or directory
Dez 06 04:36:57 robotest-52efe7d3-node-0 systemd-udevd[571]: could not unlink /run/udev/queue: No such file or directory
Dez 06 04:36:57 robotest-52efe7d3-node-0 systemd-udevd[571]: could not unlink /run/udev/queue: No such file or directory
...
```
Not sure if these are related though - need more time to look into this.

Reply:
I don't think these affect anything (I get the same warnings on host) but this actually reminded me that udevd should not be running inside planet. Previously it wasn't launched b/c /sys was mounted as read-only, but now it's read-write cause OpenEBS needs it. I've masked systemd-udevd service like *iscsi* services above.